python/lsst/meas/extensions/piff/piffPsfDeterminer.py

```diff
@@ -83,7 +90,11 @@ def computeWeight(maskedImage, maxSNR):
     """
     imArr = maskedImage.image.array
     varArr = maskedImage.variance.array
-    good = (varArr != 0) & np.isfinite(varArr) & np.isfinite(imArr)
+    good = (varArr != 0)
+    good &= np.isfinite(varArr)
+    good &= np.isfinite(imArr)
+    bitmask = maskedImage.mask.getPlaneBitMask(zeroWeightMaskBits)
+    good &= ((maskedImage.mask.array & bitmask) == 0)
```

Comment:
Very minor, but this could be ...
```
bitmask = maskedImage.mask.getPlaneBitMask(zeroWeightMaskBits)
good = ((varArr != 0)
        & (np.isfinite(varArr))
        & (np.isfinite(imArr))
        & ((maskedImage.mask.array & bitmask) == 0)
       )
```